dev-docs/bidders/consumable-server.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: bidder
+title: Consumable
+description: Prebid Consumable Bidder adapter
+userIds: all
+pbjs: true
+pbs: true
+pbs_app_supported: true
+schain_supported: true
+coppa_supported: true
+tcfeu_supported: true
+floors_supported: true
+usp_supported: true
+gpp_supported: true
+biddercode: consumable
+media_types: banner, video, audio
+gvl_id: 591
+deals_supported: true
+fpd_supported: true
+sidebarType: 1
+---
+
+### Note
+
+The Consumable adapter requires setup and approval from your Consumable account manager, even for existing Consumable publishers. Please reach out to your account manager to enable Prebid for your account.
+
+### Disclosure
+
+The Consumable bid adapter may cycle the ad initially shown with a new one at various intervals. This means the advertiser meta-data accompanying the bid response may be incomplete at the time of response.
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                    | Example | Type      |
+|-------------|----------|--------------------------------|---------|-----------|
+| `siteId`    | required | The site ID from Consumable.    | `12345` | `integer` |
+| `networkId` | required | The network ID from Consumable. | `9969`  | `integer` |
+| `unitId` | required | The unit ID from Consumable. | `987654`  | `integer` |
+| `unitName` | required | The unit name from Consumable. | `cnsmbl-unit`  | `string` |
+| `placementId` | required | the placementid from Consumable. | `0421008445828ceb46f496700a5fa65e` | `string` |
+
+## Table of contents
+
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Supported media types](#supported-media-types)
+* [Setup instructions to call Consumable through Prebid Server](#setup-instructions-to-call-consumable-through-prebid-server)
+  * [Call Consumable from a web browser](#call-consumable-from-a-web-browser)
+  * [Call Consumable from Prebid Mobile SDK](#call-consumable-from-prebid-mobile-sdk)
+  * [Call Consumable from CTV/long-form video environment](#call-consumable-from-ctvlong-form-video-environment)
+* [Bid request parameters](#bid-request-parameters)
+  * [Site](#site)
+  * [App](#app)
+* [Examples](#examples)
+
+<a id="introduction"></a>
```

Comment:
```suggestion
```

All headings can be targeted and thus no additional, empty anchor tags are needed